slither.db.json

```diff
@@ -1 +1 @@
70, 471, 472, 473, 474, 475, 476, 477, 478, 479, 480, 481, 482, 483, 484, 485, 486, 487, 488, 489, 490, 491, 492, 493, 494, 495, 496, 497, 498, 499, 500, 501, 502, 503, 504, 505, 506, 507, 508, 509, 510, 511, 512, 513, 514, 515, 516, 517, 518, 519, 520, 521, 522, 523, 524, 525, 526, 527, 528, 529, 530, 531, 532, 533, 534, 535, 536, 537, 538, 539, 540, 541, 542, 543, 544, 545, 546, 547, 548, 549, 550, 551, 552, 553, 554, 555, 556, 557, 558, 559, 560, 561, 562, 563, 564, 565, 566, 567, 568, 569, 570, 571, 572, 573, 574, 575, 576, 577, 578, 579, 580, 581, 582, 583, 584, 585, 586, 587, 588, 589, 590, 591, 592, 593, 594, 595, 596, 597, 598, 599, 600, 601, 602, 603, 604, 605, 606, 607, 608, 609, 610, 611, 612, 613, 614, 615, 616, 617, 618, 619, 620, 621, 622, 623, 624, 625, 626, 627, 628, 629, 630, 631, 632, 633], "starting_column": 1, "ending_column": 2}}, "signature": "deconstructCall(bytes)"}}], "additional_fields": {"contains_assembly": true}}, {"check": "constant-function", "impact": "Medium", "confidence": "Medium", "description": "ColonyTask.deconstructRoleChangeCall (contracts/ColonyTask.sol#574-580) is declared view but contains assembly code\n", "elements": [{"type": "function", "name": "deconstructRoleChangeCall", "source_mapping": {"start": 20120, "length": 303, "filename_used": "/Users/Elena/colonyNetwork/contracts/ColonyTask.sol", "filename_relative": "contracts/ColonyTask.sol", "filename_absolute": "/Users/Elena/colonyNetwork/contracts/ColonyTask.sol", "filename_short": "contracts/ColonyTask.sol", "lines": [574, 575, 576, 577, 578, 579, 580], "starting_column": 3, "ending_column": 4}, "type_specific_fields": {"parent": {"type": "contract", "name": "ColonyTask", "source_mapping": {"start": 792, "length": 21629, "filename_used": "/Users/Elena/colonyNetwork/contracts/ColonyTask.sol", "filename_relative": "contracts/ColonyTask.sol", "filename_absolute": "/Users/Elena/colonyNetwork/contracts/ColonyTask.sol", "filename_short": "contracts/ColonyTask.sol", "lines": [24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98, 99, 100, 101, 102, 103, 104, 105, 106, 107, 108, 109, 110, 111, 112, 113, 114, 115, 116, 117, 118, 119, 120, 121, 122, 123, 124, 125, 126, 127, 128, 129, 130, 131, 132, 133, 134, 135, 136, 137, 138, 139, 140, 141, 142, 143, 144, 145, 146, 147, 148, 149, 150, 151, 152, 153, 154, 155, 156, 157, 158, 159, 160, 161, 162, 163, 164, 165, 166, 167, 168, 169, 170, 171, 172, 173, 174, 175, 176, 177, 178, 179, 180, 181, 182, 183, 184, 185, 186, 187, 188, 189, 190, 191, 192, 193, 194, 195, 196, 197, 198, 199, 200, 201, 202, 203, 204, 205, 206, 207, 208, 209, 210, 211, 212, 213, 214, 215, 216, 217, 218, 219, 220, 221, 222, 223, 224, 225, 226, 227, 228, 229, 230, 231, 232, 233, 234, 235, 236, 237, 238,
```

Comment:
Does this file need to be in source control? Can we `.gitignore` it?

Reply:
It is intentionally under source control. This is the db produced by `slither` containing all explicitly ignored  errors and warnings from our codebase.